src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentValidator.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Provides a default implementation of <see cref="IControllerActionArgumentBinder"/>.
+    /// Uses ModelBinding to populate action parameters.
+    /// </summary>
+    public class DefaultControllerActionArgumentValidator : IControllerActionArgumentValidator
+    {
+        private readonly IModelMetadataProvider _modelMetadataProvider;
+        private readonly IValidationExcludeFiltersProvider _options;
+
+        public DefaultControllerActionArgumentValidator(
+            IModelMetadataProvider modelMetadataProvider,IValidationExcludeFiltersProvider excludeFilters)
+        {
+            _modelMetadataProvider = modelMetadataProvider;
+            _options = excludeFilters;
+        }
+
+        public Task ValidateArgumentsAsync([NotNull]ActionContext actionContext, [NotNull]ActionBindingContext bindingContext, [NotNull]IDictionary<string, object> actionArguments)
```

Comment:
Similar to DefaultControllerActionArgumentBinder. its a method on the interface.